web/src/views/Search/Search.vue

```diff
@@ -89,6 +113,15 @@ export default Vue.extend({
     <sidebar />
     <v-main v-if="typeResults">
       <v-container fluid>
+        <v-row>
+          <v-col :cols="12">
+            <ChartContainer>
```

Comment:
@subdavis I followed the pattern in another visualization (DateHistogram/Histogram2) here for ChartContainer and how to listen for resizes in the UpSet component. Am I using it as intended here? It's a very nice piece of machinery for helping update vis on resize and it seems to work.

Reply:
Looks good to me!